_layouts/download.html

```diff
@@ -149,20 +149,25 @@ <h3>Instructions</h3>
 						</div>
 						<div class="digital-store-name">Steam</div>
 					</a>
+					<a class="btn btn-digital-store" href="https://store.epicgames.com/en-US/p/godot-engine" title="Godot Engine on Epic Games Store">
```

Comment:
The link also works without the `/en-US/` part, so it might be better to let them do the redirect to the user locale?

I'll ask about the link just in case.

Reply:
Yeah, if it works universally, it's better to remove it.

```suggestion
					<a class="btn btn-digital-store" href="https://store.epicgames.com/p/godot-engine" title="Godot Engine on Epic Games Store">
```

Reply:
They confirmed that `https://store.epicgames.com/p/godot-engine` is the correct link to share 👍